accounting/receivables/customer_payments/batch.rst

```diff
@@ -0,0 +1,96 @@
+==================================================
+Batch Payments: Batch Deposits (checks, cash etc.)
+==================================================
+
+A **Batch Deposit** groups multiple payments in a single batch. This allows you to deposit multiple
+payments into your bank account with a single transaction. This is particularly useful to deposit
+cash and checks.
+
+This feature allows you to list several customer payments, and print a **deposit slip**. This ticket
+contains the details of the transactions and a reference to the batch deposit. This reference can
+be selected during a bank reconciliation to match the single bank statement line with all the
+transactions listed in the batch deposit.
+
+Configuration
+=============
+
+To activate the feature, go to :menuselection:`Accounting --> Configuration --> Settings -->
+Customer Payments`, activate **Batch Payments**, and click on *Save*.
+
+Payment Method Types
+--------------------
+
+To record new payments as part of a Batch Deposit, the Journal on which you record them must be
+correctly configured.
+
+To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, open the Journal
+you wan to edit, click on *Edit*, and open the *Advanced Settings* tab. In the *Payment Method
+Types* section, enable **Batch Deposit**, and click on Save.
+
+.. image:: media/batch-journal-settings.png
+   :align: center
+   :alt: Payment Method Types in a journal's advanced settings
+
+.. note::
+   Your main bank accounts are automatically configured to process batch payments when you activate
+   the feature.
+
+Deposit multiple payments in batch
+==================================
+
+Record payments to deposit in batch
+-----------------------------------
+
+:doc:`Register the payments <recording>` on the bank account on which you plan to deposit them, by
+opening the Customer Invoice for which you received a payment, and clicking on *Register Payment*.
+There, select the appropriate Journal linked to your bank account and select *Batch Deposit* as
+Payment Method.
+
+.. image:: media/batch-register-payment.png
+   :align: center
+   :alt: Registering a customer payment as part of a Batch Deposit in Odoo Accounting
+
+Do this step for all checks or payments you want to process in batch.
+
+.. note::
+   Make sure to write the payment reference in the **Memo** field.
+
+Add payments to a Batch Deposit
+-------------------------------
+
+:menuselection:`Accounting --> Customers --> Batch Payments`, and click on *Create*. Next, select
+the Bank and Payment Method, then click on *Add a line*.
+
+.. image:: media/batch-new-inbound.png
+   :align: center
+   :alt: Filling out a new Inbound Batch Payment form on Odoo Accounting
+
+Select all payments to include in the current Batch Deposit, and click on *Select*. You can also
+record a new payment and add it to the list by clicking on *Create*.
```

Comment:
Yes well one is a verb and the other is also a verb but as a button. I can only hope no one will complain about this unbearable repetition of the word "select". I may address this issue tomorrow if I can't find sleep thinking of it, tonight.
Cheers :p
😵

Reply:
Instead of thinking about "select", you should think about the way you just mocked me when I simply try to improve your content.
In my opinion, it's inappropriate.

As you said, "cheers"!